x-pack/plugins/alerting/server/lib/transform_action_params.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Handlebars from 'handlebars';
```

Comment:
Can we use `mustache` instead? Handlebars is "more powerful" but relies on dynamic code execution and has had two XSS vulnerabilities in the past few months.

Reply:
Why don't we just start with a rudimentary regex swap and skip the dependency altogether? We can always add it on to enable more advanced templating, but the driver here was to allow folks to turn basic templates like `Threshold reached: {threshold}%` into `Threshold reached: 95%`

Reply:
One concern I see by doing our own regex templating is that the limitations may be reached very soon. I'm thinking there may a scenario we iterate over data and such:

```
Here's a list of servers that are down:

{{#servers}}
  <li>{{name}}</li>
{{/items}}
```

Reply:
Yeah, I think mustache is the way to go personally.  People will almost certainly have more complicated needs than just simple regex replacement will support.

Reply:
Good points